.github/workflows/daily-linkcheck.yml

```diff
@@ -0,0 +1,21 @@
+name: Daily link check
+on:
+  pull_request:
+  schedule:
+    - cron: '00 19 * * *' # every day at 19:00 UTC
+
+  workflow_dispatch:
+    # Manual trigger
+
+jobs:
+  link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@main
+        with: 
+          fetch-depth: 0
+
+      - name: Run linkcheck
+        working-directory: "docs"
+        run: make linkcheck-daily
```

Comment:
Files usually should end with an empty line - you see the red sign which indicates it does not have one.